packages/credentials/.mocharc.json

```
@@ -0,0 +1,3 @@
+{
+    "enable-source-maps": true
```

Ideally would use `.mocharc.json` consistent with the other 6 packages:

```suggestion
    "enable-source-maps": true,
    "exit": true,
    "spec": ["__tests__/**/*.spec.js"]
```